src/download/linux.md

```diff
@@ -19,6 +19,11 @@ eleventyNavigation:
         <a class="button size-large" href="https://github.com/PrismLauncher/PrismLauncher/releases/download/{{version.current}}/PrismLauncher-Linux-x86_64.AppImage">Download (AppImage)</a>
         <a class="button size-large" href="https://github.com/PrismLauncher/PrismLauncher/releases/download/{{version.current}}/PrismLauncher-Linux-Qt6-Portable-{{version.current}}.tar.gz">Download Portable (tar.gz)</a>
         <a class="button size-large" href="https://github.com/PrismLauncher/PrismLauncher/releases/download/{{version.current}}/PrismLauncher-Linux-Qt5-Portable-{{version.current}}.tar.gz">Download Portable (Qt 5, tar.gz)</a>
+       <br />
+       <div class="center notification type-info">
+         <h2>Qt 5 packages will be going away soon 😔</h2>
+         <p>Support will be removed after 9.x. Users are encouraged to update to the Qt 6 builds listed below</p>
```

Comment:
this says "below", but refers to all packages including the downloads above, doesn't it? Felt a bit odd while reading through the page. Maybe just omit that word? Or replace it with "here" or something

Reply:
I think "here" implies only the download buttons right next to this box. Omitting it would probably be better